static/pattern-library/sass/utilities/_variables.scss

```diff
@@ -17,6 +17,8 @@
 // ----------------------------
 $em-base: 16; // deliberately sets bourbon-based em-base (http://bourbon.io/docs/#em-base)
 
+$static-path: '..' !default;
```

Comment:
NOTE: We'll need to sync this change up with the actual UXPL source code ``_variables.scss`` partial - https://github.com/edx/ux-pattern-library/blob/master/_src/pattern-library/sass/utilities/_variables.scss.